examples/light-switch-app/nrfconnect/main/BindingHandler.cpp

```diff
@@ -147,11 +203,25 @@ void BindingHandler::InitInternal(intptr_t arg)
     PrintBindingTable();
 }
 
+bool BindingHandler::IsGroupBound()
+{
+    BindingTable & bindingTable = BindingTable::GetInstance();
+
+    for (auto & entry : bindingTable)
```

Comment:
That would fit into [std::find_if](https://en.cppreference.com/w/cpp/algorithm/find) semantics, not sure however how `#include <algorithm>` would impact the flash.

Reply:
I checked that, and BindingTable has no Iterator, which allows using std::find_if semantics. I'm also worried about the flash impact. I guess the "for" function isn't so extensive here.

Reply:
I think it must have interators implemented to be used in for range loop :)

https://github.com/project-chip/connectedhomeip/blob/master/src/app/util/binding-table.h#L38